shell/packages/sandstorm-db/migrations.js

```diff
@@ -312,6 +312,50 @@ function repairEmailIdentityIds() {
   });
 }
 
+function splitAccountUsersAndIdentityUsers() {
+  Meteor.users.find({"identities": {$exists: true}}).forEach(function (user) {
+    if (user.identities.length != 1) {
```

Comment:
The query parameter  `{"identities": {$exists: true}` implies that such users will be passed over on the second run.